folium/features.py

```diff
@@ -91,7 +91,8 @@ def render(self, **kwargs):
                                             'if it is not in a Figure.')
 
         figure.header.add_child(
-            JavascriptLink('https://cdnjs.cloudflare.com/ajax/libs/leaflet-dvf/0.3.0/leaflet-dvf.markers.min.js'),
+            JavascriptLink(
+                'https://cdnjs.cloudflare.com/ajax/libs/leaflet-dvf/0.3.0/leaflet-dvf.markers.min.js'),
```

Comment:
Please do not change the current layout of the code. It makes it harder to review.
We already have a `noqa` to keep this in a single line.

Reply:
Hi @ocefpaf, sorry about that! I just realized after I made the commit. It was automatic code format by the text editor because of the length of the line.
Shall I do a new commit undoing that or amend my local copy and push again?

Reply:
> Hi @ocefpaf, sorry about that! I just realized after I made the commit. It was an automatic code format by the text editor because of the length of the line.

No problem. I do that all the time.

> Shall I do a new commit undoing that or amend my local copy and push it again?

Whatever is easier for you. We can squash it later before merging.

Reply:
I pushed again with only my  additions.
Cheers!